assets/common.sh

```diff
@@ -93,7 +93,7 @@ start_docker() {
   declare -fx try_start
   trap stop_docker EXIT
 
-  if ! timeout -t ${STARTUP_TIMEOUT} bash -ce 'while true; do try_start && break; done'; then
```

Comment:
There *might* be a reason why we need this `-t` 🤔 

From [`man 1 timeout`](http://man7.org/linux/man-pages/man1/timeout.1.html) it indeed says that we don't need, but it might be that busybox (what alpine uses as a replacement for these binaries) needs.

Reply:
Oh yeah, busybox:

```sh
BusyBox v1.29.3 (2019-01-24 07:45:07 UTC) multi-call binary.

Usage: timeout [-t SECS] [-s SIG] PROG ARGS

Runs PROG. Sends SIG to it if it is not gone in SECS seconds.
Defaults: SECS: 10, SIG: TERM.
```

To get rid of that difference we can install `coreutils` (the package that `timout` lives in which `busybox` "immitates"):

```sh
apk add --update coreutils
timeout --help
Usage: timeout [OPTION] DURATION COMMAND [ARG]...
```

but that *might* be too much 🤔

Reply:
Yeah I remember seeing that `busybox` requires a separate flag for the duration whereas the `ubuntu` image just had it as a required arg.

Reply:
Oh I just realized what you were talking about, this change was originally made under the assumption that we will only be using it to build `ubuntu` images. However we have since changed that to build both `alpine` and `ubuntu` images and so this will break things.